content/docs/addons-test-utils.md

```diff
@@ -103,26 +103,26 @@ afterEach(() => {
   container = null;
 });
 
-it('can render and update a counter', () => {
-  // Test first render and componentDidMount
+it('sayacı render edebilir ve güncelleyebilir', () => {
+  // render ve componentDidMount'u test eder
   act(() => {
     ReactDOM.render(<Counter />, container);
   });
   const button = container.querySelector('button');
   const label = container.querySelector('p');
-  expect(label.textContent).toBe('You clicked 0 times');
-  expect(document.title).toBe('You clicked 0 times');
+  expect(label.textContent).toBe('0 kez tıkladınız');
+  expect(document.title).toBe('0 kez tıkladınız');
 
-  // Test second render and componentDidUpdate
+  // render and componentDidUpdate'u test eder
   act(() => {
     button.dispatchEvent(new MouseEvent('click', {bubbles: true}));
   });
-  expect(label.textContent).toBe('You clicked 1 times');
-  expect(document.title).toBe('You clicked 1 times');
+  expect(label.textContent).toBe('1 kez tıkladınız');
+  expect(document.title).toBe('1 kez tıkladınız');
 });
 ```
 
-Don't forget that dispatching DOM events only works when the DOM container is added to the `document`. You can use a helper like [`react-testing-library`](https://github.com/kentcdodds/react-testing-library) to reduce the boilerplate code.
+DOM konteynerı `document`e eklendiğinde DOM olaylarının gönderilmesini sağlayabilirsiniz. Ana proje iskeletinizi küçültmek için [`react-testing-library`](https://github.com/kentcdodds/react-testing-library) gibi bir araç kullanabilirsiniz
```

Comment:
Burada `sağlamak`eylemi yerine aslında bir uyarı var. Aşağıdaki gibi bir cümle nasıl olur sence?    
  
Unutmayınız ki,  `document`'a DOM `container`'ı eklenmediği sürece, `dispatchEvent()` ile DOM olaylarının ilgili elemana gönderilmesi **çalışmayacaktır**. Bu gibi angarya kodlarla uğraşmamak için, [`react-testing-library`](https://github.com/kentcdodds/react-testing-library) gibi bir kütüphaneyi kullanabilirsiniz.

Daha kısa hali:  
Unutmayınız ki,  `document`'a DOM `container`'ı eklenmediği sürece, `dispatchEvent()` **çalışmayacaktır**. Bu gibi angarya kodlarla uğraşmamak için, [`react-testing-library`](https://github.com/kentcdodds/react-testing-library) gibi bir kütüphaneyi kullanabilirsiniz.